File: Menu_app/app/src/main/java/com/example/menu_app/classes/dish.java
package com.example.menu_app.classes;

public class dish {
    private String name;            //name of the dish
    private String cn_name;         //chinese name of the dish
    private String staff_name;      //staff name of the dish
    private String id;              //the id of the dish (some dishes won't have any)
    private int price;           //price of the dish
    private int dish_counter;       //number of times the dish has been ordered

    public dish(String name, String cn_name, String staff_name, String id, int price, int dish_counter){
        this.name = name;
        this.cn_name = cn_name;
        this.staff_name = staff_name;
        this.id = id;
        this.price = price;
        this.dish_counter = 0;
    }
    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getCn_name() {
        return cn_name;
    }

    public void setCn_name(String cn_name) {
        this.cn_name = cn_name;
    }

    public String getStaff_name() {
        return staff_name;
    }

    public void setStaff_name(String staff_name) {
        this.staff_name = staff_name;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public double getPrice() {
        return price/100;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public int getDish_counter() {
        return dish_counter;
    }

    public void setDish_counter(int dish_counter) {
        this.dish_counter = dish_counter;
    }
}

File: app/src/main/java/com/example/menu_app/classes/dish.kt
package com.example.menu_app.classes

class dish {
    lateinit var name: String
    lateinit var cn_name: String
    lateinit var staff_name: String
    lateinit var id: String
    var price: Double = 0.0
}
fun getName(dish: dish): String {
    return dish.name
}

fun setName(dish: dish, name: String) {
    dish.name = name
}

fun getCn_name(dish: dish): String {
    return dish.cn_name
}

fun setCn_name(dish: dish, cn_name: String) {
    dish.cn_name = cn_name
}

fun getStaff_name(dish: dish): String {
    return dish.staff_name
}

fun setStaff_name(dish: dish, staff_name: String) {
    dish.staff_name = staff_name
}

fun getId(dish: dish): String {
    return dish.id
}

fun setId(dish: dish, id: String) {
    dish.id = id
}

fun getPrice(dish: dish): Double {
    return dish.price
}

fun setPrice(dish: dish, price: Double) {
    dish.price = price
}
